lightning/src/events/mod.rs

```diff
@@ -1078,13 +1082,15 @@ impl MaybeReadable for Event {
 					let mut prev_channel_id = None;
 					let mut claim_from_onchain_tx = false;
 					let mut next_channel_id = None;
+					let mut amount_forwarded_msat = None;
 					read_tlv_fields!(reader, {
 						(0, fee_earned_msat, option),
 						(1, prev_channel_id, option),
 						(2, claim_from_onchain_tx, required),
 						(3, next_channel_id, option),
+						(4, amount_forwarded_msat, option),
```

Comment:
I believe the type number when reading should match the one you just changed when writing in `impl Writeable` :)